examples/tests/igraph_eulerian_paths.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include "test_utilities.inc"
+
+int main() {
+
+    igraph_t graph, graph2, graph3, graph4, graph5, graph6, graph7, graph8, graph9;
```

Comment:
You can use a single variable `graph` which you destroy as soon as you are done with one graph, and create a new graph in its place. It is not necessary to have 9 different ones.

Reply:
This got solved in the meanwhile.